lib/modules/dosomething/dosomething_api/resources/reportback_resource.inc

```diff
@@ -133,14 +154,12 @@ function _reportback_resource_index($campaigns, $status, $count, $random, $page,
     'page' => $page,
     'load_user' => $load_user,
     'flagged' => $flagged,
-  );
+  ];
 
-  $reportbacks = new ReportbackTransformer;
-  return $reportbacks->index($parameters);
+  return (new ReportbackTransformer)->index($parameters);
```

Comment:
Ah, yeah. That's fine, not a merge blocker.

Longer term, this is the kind of thing we'd probably want to pull from an service container anyways if we're greenlit to keep working on the prototype started in #5932. Something like..

```php
return app('\Phoenix\Reportbacks\ReportbackTransformer')->index($parameters);
```